test/lib/finest/nif.ex

```diff
@@ -56,5 +56,12 @@ defmodule Finest.NIF do
   def shared_mutex_unique_lock_test(), do: err!()
   def shared_mutex_shared_lock_test(), do: err!()
 
+  def compare_eq_test(_lhs, _rhs), do: err!()
```

Comment:
Super nitpick, but let's remove `_test` suffix here, since the NIF just applies an operation.

It makes sense in the few NIFs above, because the actual test logic is inside the NIF :)